### ISSUE TITLE
Incorrect MU changes after installs

### ISSUE THREAD

**Issue by OtisF** [mtgred/netrunner#7121]
I just had this happen to me in two separate jnet games, after installing a program with Arissana's ability, I was gifted with extra MU. The only common factors between both occurrences seem to be that I performed the install mid-run, and that I was prompted to trash another program to make room (which I did so).

Game 1, turn 12, click 2. Running HQ, installed a Slap Vandal onto a Magnet, trashed a Kyuban on another server to make room, but the newly installed SV didn't seem to increase my used MU: 

https://www.jinteki.net/replay/dd1a3000-4466-4441-8c20-432c66581286?n=107&d=0

Game 2, turn 6, click 3. Running R&D, installed a Kyuban onto a Thimblerig, trashing a Kyuban elsewhere, and after passing the ice (and Thimblerig moves) I suddenly have 2MU free:

https://www.jinteki.net/replay/39e0281c-7ad7-4359-aad8-6fe57aba7e30?n=53&d=0

Additionally, my MU in the second game eventually fixes itself, I "regain" an MU when returning the Kyuban that was on the Thimble with Art Vernissage (runner turn 8), and then a second time when Thimble moves whilst hosting another Kyuban (corp turn 12).

In conclusion, Thimblerig is a hell of a card, huh.

**Comment by NBKelly** [comment]
In both of these, I *think* it's the programs being on magnet - magnet is a bit wonky, and the mu cost of programs on it is not counted right now.
